### DISCUSSION TITLE
CodeBuddies Frontend Pairing Hangouts (Recordings from Twitch)

### DISCUSSION THREAD

**Discussion by lpatmo** [codebuddies/frontend#133] [General]
Leave notes here! Each hangout should have its own post, with recording links :)

**Comment by lpatmo** [comment]
August 18th, 2020

**Reply by lpatmo** [reply]
https://github.com/search?q=axiosMock.get.mockResolvedValueOnce&type=Code

https://jestjs.io/docs/en/mock-function-api#mockfnmockresolvedvalueoncevalue

https://www.leighhalliday.com/async-axios-react-testing-library

**Comment by watchtheblur** [comment]
useReducer usage in CBV3

https://github.com/codebuddies/frontend/blob/issue55-post/src/components/Resources/submitResource.js
